src/Runner.Worker/ActionManifestManager.cs

```diff
@@ -402,6 +415,20 @@ public ActionDefinitionData Load(IExecutionContext executionContext, string mani
                         };
                     }
                 }
+                else if (string.Equals(usingToken.Value, "composite", StringComparison.OrdinalIgnoreCase) && !String.IsNullOrEmpty(Environment.GetEnvironmentVariable("TESTING_COMPOSITE_ACTIONS_ALPHA")))
+                {
+                    if (stepsLoaded == null)
+                    {
+                        throw new ArgumentNullException($"No steps provided.");
```

Comment:
does this error give the user enough context to know where the problem is? Like does it convey that an error happened when loading the path-to-action.yml

Reply:
i guess we would have the same problem with the other place too (in this file)

Reply:
I think this error makes sense and is appropriate. 

At the top of `ConvertRuns`: 
`var stepsLoaded = default(List<Pipelines.ActionStep>);`

Then, we check in the `action.yml` file to see if there is any `steps` token:
```
case "steps":
                        if (!string.IsNullOrEmpty(Environment.GetEnvironmentVariable("TESTING_COMPOSITE_ACTIONS_ALPHA")))
                        {
                            var steps = run.Value.AssertSequence("steps");
                            var evaluator = executionContext.ToPipelineTemplateEvaluator();
                            stepsLoaded = evaluator.LoadCompositeSteps(steps);
                            break;
                        }
                        throw new Exception("You aren't supposed to be using Composite Actions yet!");
```

If there are any issues with loading the steps, there would be an error reported in the `LoadSteps()` or `LoadStep()` function in the pipeline template evaulator. 

Thus, if `stepsLoaded` is null, it must be because there was no `steps` attribute written in the `action.yml` by the user.

Thus, I believe throwing this message is appropriate.

Reply:
(Maybe add a more helpful error message + including file name, etc.  to show user that it's because of their yaml file)